src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/SHA256.cs

```diff
@@ -36,21 +30,22 @@ public Implementation()
                 HashSizeValue = _hashProvider.HashSizeInBytes * 8;
             }
 
-            protected sealed override void HashCore(byte[] array, int ibStart, int cbSize)
-            {
+            protected sealed override void HashCore(byte[] array, int ibStart, int cbSize) =>
                 _hashProvider.AppendHashData(array, ibStart, cbSize);
-            }
 
-            protected sealed override byte[] HashFinal()
-            {
-                return _hashProvider.FinalizeHashAndReset();
-            }
+            protected override void HashCore(ReadOnlySpan<byte> source) =>
```

Comment:
With regards to the passive-aggressive thumbs down, if you don't like my agree-to-disagree comment, then what exactly do you suggest we do?  You don't like the way I've suggested, and I don't like the way you've suggested.  My default is to make the entire codebase as consistent as possible, not just this small corner of it, and the rest of the codebase relies on a type being sealed to indicate that its methods are also sealed.  If someone unseals a type, they and the code reviewers of that change need to understand all of the ramifications of that, not just but including that any unsealed members become overridable... which btw is often why someone chooses to unseal a type, to override its members.